components/dashboard/Messages.tsx

```
@@ -395,7 +395,10 @@ const MessagesList = ({
                 : "bg-gray-200 text-gray-800 rounded-bl-none"
             }`}
           >
-            <p className="text-sm">{msg.content}</p>
+            <p
+              className="text-sm"
+              dangerouslySetInnerHTML={{ __html: msg.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)